docs/UserGuide.md

```diff
@@ -69,17 +69,18 @@ Format: `help`
 
 ### Adding a person: `add`
 
-Adds a person to the address book.
+Adds a resident to Hall-y.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS r/ROOM_NUMBER g/GENDER m/MATRICULATION_NUMBER`
 
-<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags (including 0)
-</div>
+```
+Tip: ROOM_NUMBER is in the format <Block>_<Room Number>, e.g. B505, C201, etc
+```
```

Comment:
Should this be a series of instructions instead of a tip?

i.e. something like the `edit` command in the UG:

- GENDER: **must be either M or F**
- ROOM_NUMBER: **must be \<Block>_\<Room Number>**

Reply:
Should the block and room number be separated? e.g. b/BLOCK r/ROOM_NUMBER
It would be easier for the user to type without the formatting error.

Reply:
@teekoksiang I think we settled on combining both to minimize the number of fields required when creating a resident. So B05 will be internally parsed as "B" and "05"